contracts/token/ERC6909/extensions/draft-ERC6909ContentURI.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ERC6909} from "../draft-ERC6909.sol";
+import {IERC6909ContentURI} from "../../../interfaces/draft-IERC6909.sol";
+
+contract ERC6909ContentURI is ERC6909, IERC6909ContentURI {
+    string private _contractURI;
+    mapping(uint256 id => string) private _tokenURIs;
+
+    function contractURI() external view virtual override returns (string memory) {
+        return _contractURI;
+    }
+
+    function tokenURI(uint256 id) external view virtual override returns (string memory) {
+        return _tokenURIs[id];
+    }
```

Comment:
I'm confused about this since the standard states:

> MUST replace occurrences of {id} in the returned URI string by the client.

I don't understand whether that means the contract must replace `{id}` in the URI string

Reply:
It's coping ERC1155. It's done off-chain.